filebeat/input/input.go

```diff
@@ -0,0 +1,150 @@
+package input
```

Comment:
I assume that code is mostly copy/paste/rename

Reply:
Exact, this should be as is.